scripts/custom/clean_comparatedNumber.py

```diff
@@ -0,0 +1,31 @@
+from scripts import utils
+import re 
+
+comparator = [
+    "<=",
+    ">=",
+    "=",
+    "=>",
+    "=<",
+    ">",
+    "<",
+    "",
+]
+
+def clean_comparatedNumber(raw_input):
+    """Remove the comparator sign from a comparated number ("=>16") to ("16")
+    """
+
+    if utils.is_empty(raw_input):
+        return None
+    
+    comparatedNumber=re.match(r"([<=>]?[=<>=]?)(.*)",raw_input)
```

Comment:
You can use the regexp on clean_quantity to defined what a number looks like (instead of .*) :
re.search(r"-?(0|[1-9][0-9]*)(\.[0-9]+)?([eE][+-]?[0-9]+)?", raw_input)
Something like: ([<>]?=?)(-?(0|[1-9][0-9]*\.[0-9]+)?([eE][+-]?[0-9]+)?) maybe